react/components/EditorContainer/StoreEditor/Styles/StyleEditor/utils/typography.ts

```diff
@@ -54,3 +54,18 @@ export function getFontFlavour({
 }: FontFileInput): FontFlavour {
   return [fontStyle, fontWeight] as FontFlavour
 }
+
+export function prettify(name: string) {
```

Comment:
I think we already have [`lodash.startcase`](https://lodash.com/docs/4.17.11#startCase) installed that does pretty much the same as this function. Could you check if the `lodash` method works for your use?

Reply:
Works.